src/env.d/09-libc

```diff
@@ -0,0 +1 @@
+LIBC_VERSION=$(/lib"$LIB_SUFFIX"/libc.so.6 2>/dev/null | awk 'match($0, /Gentoo ([^-]+)/) {print substr($0, RSTART+7, RLENGTH-7)}')
```

Comment:
What are `RSTART` and `RLENGTH`?  I don't see where they are set.

Reply:
Those are awk commands. They don't need to be set. This entire line is in `install.sh`.